test/puppetlabs/http/client/async_unbuffered_test.clj

```diff
@@ -144,7 +145,7 @@
        (let [response @(common/get client (str "http://localhost:" 12345 "/bad") opts)
              {:keys [error]} response]
          (is error)
-         (is (instance? ConnectException error)))))))
+         (is (instance? ConnectionClosedException error)))))))
```

Comment:
This seems like a weird change, do you know what instigated it?

Reply:
A change in commons-httpclient. The exception thrown is now `ConnectionClosedException` - this test did not work properly without this change. I'm assuming we updated the commons-httpclient version in clj-parent at some point but did not rerun the tests in this project?